### PR TITLE
Client Request Cached Methods List from JIT Server

### PR DESCRIPTION
**Pull request by luke-li-2003**
Added an option that makes the client request a list of cached methods from the server on bootstrap, and set the count for those methods as scount to improve rampup

### CODE REVIEW

**Comment by luke-li-2003** [comment]
@mpirvu

File: compiler/control/OMROptions.cpp [hunk 1]
```diff
@@ -1082,6 +1082,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"regmap",             "C\tgenerate GC maps with register maps", SET_OPTION_BIT(TR_RegisterMaps), "F", NOT_IN_SUBSET},
    {"reportEvents",       "C\tcompile event reporting hooks into code", SET_OPTION_BIT(TR_ReportMethodEnter | TR_ReportMethodExit)},
    {"reportMethodEnterExit", "D\treport method enter and exit",                   SET_OPTION_BIT(TR_ReportMethodEnter | TR_ReportMethodExit), "F"},
+   {"requestJITServerCachedMethods", "O\trequest the list of methods cached at the server on bootstrap, and prioritise the compilation requests for them", SET_OPTION_BIT(TR_RequestJITServerCachedMethods), "F"},
```
Comment (mpirvu) [inline compiler/control/OMROptions.cpp:1085]: Please add "NOT_IN_SUBSET" like in other examples seen in this file. This means that the option should not be applied to specific methods, but rather to all methods.

**Comment by mpirvu** [comment]
Please rebase your changes.

**Review by dsouzai — APPROVED** [review]

**Comment by dsouzai** [comment]
jenkins build all

**Pull request merged**